Examples/VoiceMemos/VoiceMemos/VoiceMemos.swift

```diff
@@ -234,13 +239,12 @@ struct VoiceMemosView: View {
                 Text($0)
                   .font(Font.body.monospacedDigit().bold())
                   .foregroundColor(.white)
-                  .colorMultiply(Int(duration).isMultiple(of: 2) ? .red : .black)
+                  .colorMultiply(Color(Int(duration).isMultiple(of: 2) ? .systemRed : .label))
                   .animation(.easeInOut(duration: 0.5))
               }
             }
           }
           .padding()
-          .animation(Animation.easeInOut(duration: 0.3))
```

Comment:
Lost the animation that brought the label in...not sure the correct way to bring it back, though.